lib/grape/dsl/request_response.rb

```diff
@@ -131,6 +131,11 @@ def rescue_from(*args, &block)
           namespace_stackable(:rescue_options, options)
         end
 
+        def ensure_block(&ensured_block)
```

Comment:
method description and example use are missing

Reply:
Thanks! Will address!